website/docs/r/workflow.html.markdown

```diff
@@ -63,38 +39,86 @@ resource "newrelic_workflow" "foo" {
-  * DONT_NOTIFY_FULLY_MUTED_ISSUES - Do not send notifications for fully muted issues.
-  * DONT_NOTIFY_FULLY_OR_PARTIALLY_MUTED_ISSUES - Do not send notifications for fully or partially muted issues.
-  * NOTIFY_ALL_ISSUES - Always send notifications.
-* `destination` - (Required) A nested block that contains a channel id.
-* `issues_filter` - (Required) The issues filter.  See [Nested issues_filter blocks](#nested-issues_filter-blocks) below for details.
-* `enrichments` - (Optional) A nested block that describes a workflow's enrichments. See [Nested enrichments blocks](#nested-enrichments-blocks) below for details.
+* `issues_filter` - (Required) A filter used to identify issues handled by this workflow. See [Nested issues_filter blocks](#nested-issues_filter-blocks) below for details.
+* `muting_rules_handling` - (Required) How to handle partially muted issues. See [Muting Rules](#muting-rules) below for details.
+* `destination` - (Required) Notification configuration. See [Nested destination blocks](#nested-destination-blocks) below for details.
+* `account_id` - (Optional) Determines the New Relic account in which the workflow is created. Defaults to the account defined in the provider section.
+* `enrichments_enabled` - (Optional) Whether enrichments are enabled. Defaults to true.
+* `destinations_enabled` - (Optional) **DEPRECATED** Whether destinations are enabled. Please use `enabled` instead:
+these two are different flags, but they are functionally identical. Defaults to true.
+* `enabled` - (Optional) Whether workflow is enabled. Defaults to true.
+* `enrichments` - (Optional) Workflow's enrichments. See [Nested enrichments blocks](#nested-enrichments-blocks) below for details.
 
 ### Nested `issues_filter` blocks
 
-Each workflow type supports a set of arguments for the `issues_filter` block:
+Issue filter defines how to identify issues that should be handled by the workflow. 
+It consists of a number of predicates. 
+
+An issue must match **all** predicates to be handled by the workflow.
 
-* `name` - the filter's name.
-* `type` - the filter's type.   One of: `FILTER` or `VIEW`.
-* `predicate`
-  * `attribute` - A predicate's attribute.
-  * `operator` - A predicate's operator. One of: `CONTAINS`, `DOES_NOT_CONTAIN`, `DOES_NOT_EQUAL`, `DOES_NOT_EXACTLY_MATCH`, `ENDS_WITH`, `EQUAL`, `EXACTLY_MATCHES`, `GREATER_OR_EQUAL`, `GREATER_THAN`, `IS`, `IS_NOT`, `LESS_OR_EQUAL`, `LESS_THAN` or `STARTS_WITH` (workflows).
-  * `values` - A list of values.
+For example, we have a workflow with two predicates in the issue filter: 
+- `tag.team EXACTLY_MATCHES my_team` meaning "the issue has a tag called `team` and the tag value is set to 'my_team'" 
```

Comment:
nit: tags.team

Reply:
Not unless our UI somehow changes `tags` to `tag` when showing the filter. I'll double-check, but it probably does not.